### PR TITLE
added history and diagnostic sections to CLI section

### PR DESCRIPTION
**Pull request by rosewms**
Tech writing project. Added **History** and **Diagnostic** section to the CLI file. 

image

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 1]
```diff
@@ -45,3 +45,26 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+Diagnostics
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/cli.rst:49]: Maybe call this Troubleshooting ?

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 2]
```diff
@@ -45,3 +45,26 @@ which is configured with a carefully selected set of options. This behavior
...
+-----------
+
+Use the ``--debug`` option to review debug information when running queries:
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/cli.rst:52]: to view debug information when using the CLI:

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 3]
```diff
@@ -45,3 +45,26 @@ which is configured with a carefully selected set of options. This behavior
...
+ 	Query 20181103_201856_00022_te3wy failed: 
+ 	line 1:22: Table tpch.sf1.foo does not exist io.prestosql.sql.analyzer.SemanticException: 
+ 	line 1:22: Table tpch.sf1.foo does not exist 
+ 	... 
+ 	at java.lang.Thread.run(Thread.java:748)
+
+History
+-------
+
+The CLI keeps a history of your previously used commands. You can access your history by scrolling or searching. Use the up and down arrows to scroll and Ctrl-S and Ctrl-R to search. To execute a query again, press **Enter**.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/cli.rst:67]: Reformat paragraph to 80 char width please

Comment (mosabua) [reply presto-docs/src/main/sphinx/installation/cli.rst:67]: and maybe use Control-S and Control-R and bold both as well

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 4]
```diff
@@ -45,3 +45,26 @@ which is configured with a carefully selected set of options. This behavior
...
+
+By default, you can locate the Presto history file in *~/.presto_history*.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/cli.rst:69]: double backticks for the path instead of *

Comment (rosewms) [reply presto-docs/src/main/sphinx/installation/cli.rst]: Just saw this and other two comments from 35 minutes ago... I was in the file editing it and submit before I realized there were additional comments beyond the first two you made.

Comment (mosabua) [reply presto-docs/src/main/sphinx/installation/cli.rst]: thats fine .. you can always add another change and commit it. Before merging this we will have to rebase and squash the commits anyway..

**Comment by ebyhr** [comment]
@cla-bot check

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 5]
```diff
@@ -45,3 +45,27 @@ which is configured with a carefully selected set of options. This behavior
...
+History
+-------
+
+The CLI keeps a history of your previously used commands. Access your history by scrolling or searching. Scroll using the up or down arrows and search using **Ctrl-S** and **Ctrl-R**. To execute a query again, press **Enter**.
+
+By default, you can locate the Presto history file in ``~/.presto_history``.
+Use the ``PRESTO_HISTORY_FILE`` environment variable to change the default.
+
+Troubleshooting
+-----------
```
Comment (ebyhr) [inline presto-docs/src/main/sphinx/installation/cli.rst:59]: This is the cause of CI failure.
```suggestion
---------------
```

File: presto-docs/src/main/sphinx/installation/cli.rst [hunk 6]
```diff
@@ -45,3 +45,27 @@ which is configured with a carefully selected set of options. This behavior
...
+
+.. code-block:: none
+
+ 	$ presto --debug 
+ 	presto:sf1> SELECT count(*) FROM foo; 
+ 	Query 20181103_201856_00022_te3wy failed: 
+ 	line 1:22: Table tpch.sf1.foo does not exist io.prestosql.sql.analyzer.SemanticException: 
+ 	line 1:22: Table tpch.sf1.foo does not exist 
+ 	... 
+ 	at java.lang.Thread.run(Thread.java:748)
```
Comment (ebyhr) [inline presto-docs/src/main/sphinx/installation/cli.rst:71]: I would recommend using the actual message like:
```
 	Query 20181103_201856_00022_te3wy failed: line 1:22: Table 'tpch.sf1.foo' does not exist
 	io.prestosql.spi.PrestoException: line 1:22: Table 'tpch.sf1.foo' does not exist
 	        at io.prestosql.sql.analyzer.SemanticExceptions.semanticException(SemanticExceptions.java:48)
            ...
 	        at java.lang.Thread.run(Thread.java:748)
```

**Comment by mosabua** [comment]
@ebyhr I will be working with @rosewms to finish this PR off to be ready for merge next week.

File: presto-docs/src/main/sphinx/installation/cli.rst (hunk 3 above)
Comment (rosewms) [reply presto-docs/src/main/sphinx/installation/cli.rst]: Here's what the google dev guide recommends for keyboard elements: https://developers.google.com/style/ui-elements#keyboard

**Comment by rosewms** [comment]
Created new PR: https://github.com/prestosql/presto/pull/4694

**Pull request closed without merging**